plugins/woocommerce/src/Internal/Traits/OrderAttributionMeta.php

```diff
@@ -385,13 +385,24 @@ private function get_field_description( string $field ): string {
 	 * @return array Order count, total spend, and average spend per order.
 	 */
 	private function get_customer_history( $customer_identifier ): array {
+		/*
+		 * Exclude the statuses that aren't valid for the Customers report.
+		 * 'checkout-draft' is the checkout block's draft order status.
+		 * @see /Automattic/WooCommerce/Admin/API/Report/DataStore::get_excluded_report_order_statuses()
+		 */
+		$all_statuses_no_prefix = array_map(
+			function ( $status ) {
+				return str_replace( 'wc-', '', $status );
+			},
+			array_keys( wc_get_order_statuses() )
+		);
```

Comment:
`str_replace` can handle arrays.

```suggestion
		$all_statuses_no_prefix = str_replace( 'wc-', '', array_keys( wc_get_order_statuses() ) );
```